mne/minimum_norm/spatial_resolution.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#
+# License: BSD (3-clause)
+"""Compute resolution metrics from resolution matrix.
+
+Resolution metrics: localisation error, spatial extent, relative amplitude.
+Metrics can be computed for point-spread and cross-talk functions (PSFs/CTFs).
+"""
+import numpy as np
+
+from mne import SourceEstimate
+from mne.utils import logger, verbose
+
+
+@verbose
+def resolution_metrics(resmat, src, function, metric, threshold=0.5):
```

Comment:
I would put default values for function and metric. function to psf and metric to peak_err. It's always nicer to have functions that work with less investigations.